Source/Core/VideoCommon/PerformanceMetrics.cpp

```diff
@@ -181,12 +241,18 @@ void PerformanceMetrics::DrawImGuiStats(const float backbuffer_scale) const
 
   if (g_ActiveConfig.bShowVPS || g_ActiveConfig.bShowVTimes)
   {
-    // Position in the top-right corner of the screen.
     int count = g_ActiveConfig.bShowVPS + 2 * g_ActiveConfig.bShowVTimes;
+    float window_height = (12.f + 17.f * count) * backbuffer_scale;
+
+    // Position in the top-right corner of the screen.
     ImGui::SetNextWindowPos(ImVec2(window_x, window_y), ImGuiCond_Always, ImVec2(1.0f, 0.0f));
     ImGui::SetNextWindowSize(ImVec2(window_width, (12.f + 17.f * count) * backbuffer_scale));
     ImGui::SetNextWindowBgAlpha(bg_alpha);
-    window_x -= window_width + window_padding;
+
+    if (stack_vertically)
+      window_y += window_height + window_padding;
+    else
+      window_x -= window_width + window_padding;
```

Comment:
These commits are mainly just moving things around since the Improved Pacing allows us to get some new data points